delphi/translators/for2py/f2grfn.py

```diff
@@ -217,30 +222,31 @@ def generate_python_src(
                 Script: <pyTranslate.py>"
         )
 
-        output_list = []
-        for item in python_source:
-            if item[2] == "module":
-                module_file = f"{temp_dir}/m_{item[1].lower()}.py"
-                try:
-                    with open(module_file, "w") as f:
-                        output_list.append("m_" + item[1].lower() + ".py")
-                        f.write(item[0])
-                except IOError:
-                    assert False, f"Unable to write to {module_file}."
-            else:
-                try:
-                    with open(python_file_name, "w") as f:
-                        output_list.append(python_file_name)
-                        f.write(item[0])
-                except IOError:
-                    assert False, f"Unable to write to {python_file_name}."
+    output_list = []
+    for item in python_source:
+        if item[2] == "module":
+            module_file = f"{temp_dir}/m_{item[1].lower()}.py"
+            try:
+                with open(module_file, "w") as f:
+                    output_list.append("m_" + item[1].lower() + ".py")
```

Comment:
Would it be possible to pull the mapping from a Fortran module name to the Python file name into a separate function?  I'm worried what happens if there happens to be an already-existing file whose name begins with "m_", in which case the current approach won't work.  Having the mapping done in one place (in this new function) will make the problem easier to fix down the road.

Reply:
@skdebray I've separated out the coded and put it into its own function. Though, I feel this is like a temporary solution. I will think about how the more long-term solution that we can avoid the possible name conflict.